app/services/platforms/youtube.ts

```diff
@@ -278,6 +285,84 @@ export class YoutubeService
     this.state.liveStreamingEnabled = enabled;
   }
 
+  async createVertical(settings: IGoLiveSettings): Promise<ICustomStreamDestination> {
+    // {
+    //   id: string;
+    //   snippet: {
+    //     isDefaultStream: boolean;
+    //   };
+    //   cdn: {
+    //     ingestionInfo: {
+    //       /**
+    //        * streamName is actually a secret stream key
+    //        */
+    //       streamName: string;
+    //       ingestionAddress: string;
+    //     };
+    //     resolution: string;
+    //     frameRate: string;
+    //   };
+    //   status: {
+    //     streamStatus: TStreamStatus;
+    //   };
+    // }
+
+    const ytSettings = getDefined(settings.platforms.youtube);
+    const title = `${ytSettings.title}-vert`;
+
+    // OG:
+    // if (!streamToScheduledBroadcast) {
+    //   broadcast = await this.createBroadcast(ytSettings);
+    // } else {
+    //   assertIsDefined(ytSettings.broadcastId);
+    //   await this.updateBroadcast(ytSettings.broadcastId, ytSettings);
+    //   broadcast = await this.fetchBroadcast(ytSettings.broadcastId);
+    // }
+    // NOW:
+    const verticalBroadcast = await this.createBroadcast({ ...ytSettings, title });
+    //console.log('verticalBroadcast', JSON.stringify(verticalBroadcast, null, 2));
+
+    // OG:
+    // let stream: IYoutubeLiveStream;
+    // if (!broadcast.contentDetails.boundStreamId) {
+    //   stream = await this.createLiveStream(broadcast.snippet.title);
+    //   await this.bindStreamToBroadcast(broadcast.id, stream.id);
+    // } else {
+    //   stream = await this.fetchLiveStream(broadcast.contentDetails.boundStreamId);
+    // }
+    // NOW:
+    const verticalStream = await this.createLiveStream(verticalBroadcast.snippet.title);
+    const verticalBoundBroadcast = await this.bindStreamToBroadcast(
+      verticalBroadcast.id,
+      verticalStream.id,
+    );
+    //console.log('verticalStream', JSON.stringify(verticalStream, null, 2));
+    //console.log('verticalBoundBroadcast', JSON.stringify(verticalBoundBroadcast, null, 2));
+
+    await this.updateCategory(verticalBroadcast.id, ytSettings.categoryId!);
+
+    const verticalStreamKey = verticalStream.cdn.ingestionInfo.streamName;
+    this.SET_VERTICAL_STREAM_KEY(verticalStreamKey);
+    this.SET_VERTICAL_BROADCAST(verticalBoundBroadcast);
+    //console.log('createVertical this.state.verticalStreamKey', this.state.verticalStreamKey);
+    //console.log('createVertical verticalBroadcast', verticalBoundBroadcast);
+
+    // JUST TO CHECK:
+    //const liveBroadcast = await this.fetchLiveStream(
+    //  this.state.verticalBroadcast.contentDetails.boundStreamId,
+    //);
+    // console.log('fetched vertical', JSON.stringify(liveBroadcast, null, 2));
```

Comment:
would be good to clean up these commented segments and various console logs before merging